### PR TITLE
HPCC-16557 Create ESDL Bindings/Defintions dali path by default

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: rpastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@ghalliday please review.

File: esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp [hunk 1]
```diff
@@ -267,7 +267,7 @@ void CWsESDLConfigEx::init(IPropertyTree *cfg, const char *process, const char *
 
 IPropertyTree * CWsESDLConfigEx::getESDLDefinitionRegistry(const char * wsEclId, bool readonly)
 {
-    if (!ensureSDSPath("ESDL/Definitions"))
+	if (!ensureSDSPath(ESDL_DEFS_ROOT_PATH))
```
Comment (ghalliday) [inline esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp]: trivial: indentation messed up

**Comment by rpastrana** [comment]
@ghalliday please review latest commit.

**Comment by ghalliday** [comment]
Looks ok - @rpastrana can you squash those commits.

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**